### DISCUSSION TITLE
fun fact!!!!!!!!

### DISCUSSION THREAD

**Discussion by rwg100** [3kh0/ext-remover#310] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
if you pour gasoline on your chromebook, throw it into a house, and light yourself on fire, you are now on fire! your chromebook, is in fact, not on fire, because you didnt light it on fire!!!!
you also unenrolled yourself from the school in the process!

**Comment by Aka-but-nice** [comment, marked as answer]
Thank you this really helped me out, now im living in a shed THANK YOU SO MUCH
